builder/__tests__/plugins/statistics.test.js

```diff
@@ -0,0 +1,51 @@
+const childProcess = require('child_process');
+
+const statisticsFn = require('../../plugins/statistics')();
+
+const testFiles = {};
+
+let testMetalsmith;
+
+jest.mock('child_process');
+childProcess.exec.mockImplementation((command, callback) => {
+  callback(null, '\n  hash123,hsh,2019-06-07T12:00:00Z  \n');
+});
+
+
+beforeEach(() => {
+  testMetalsmith = {
+    _metadata: {
+      snippets: [
+        {
+          author: 'a'
+        },
+        {
+          author: 'b'
+        },
+        {
+          author: 'a'
+        }
+      ],
+      tags: {
+        tag1: [],
+        tag2: []
+      }
+    }
+  }
+});
+
+test('statistics function', done => {
+  statisticsFn(testFiles, testMetalsmith, () => {
```

Comment:
Can we call statisticsFn consistent with how it's used?

Reply:
> Can we call statisticsFn consistent with how it's used?

what do you mean? that is how it's used by metalsmith

Reply:
you don't call it `statisticsFn` when you use it